kernel/cClosure.ml

```diff
@@ -529,16 +529,11 @@ let mk_clos_vect env v = match v with
   [|mk_clos env v0; mk_clos env v1; mk_clos env v2; mk_clos env v3|]
 | v -> Array.Fun1.map mk_clos env v
 
-let transparent_ref flags = function
-| RelKey _ -> red_set flags fDELTA
-| ConstKey (c, _) -> red_set flags (fCONST c)
-| VarKey id -> red_set flags (fVAR id)
-
 let assoc_defined id env = match Environ.lookup_named id env with
 | LocalDef (_, c, _) -> c
 | LocalAssum _ -> raise Not_found
 
-let ref_value_cache env tab ref =
+let ref_value_cache env flags tab ref =
```

Comment:
```suggestion
let unfold_reference env flags tab ref =
```
then we don't need the alias at the end of the diff

Reply:
I'd prefer keeping the way it is now not to introduce unexpected coupling. It might be the case that I fiddle with this a bit more in the near future.